src/main/resources/org/jenkinsci/plugins/mesos/MesosCloud/config.jelly

```diff
@@ -92,6 +92,9 @@
                                     <f:entry title="${%Docker Image}" field="dockerImage" description="If using Docker, specify the docker image.">
                                         <f:textbox field="dockerImage" clazz="required" value="${slaveInfo.containerInfo.dockerImage}" default=""/>
                                     </f:entry>
+                                    <f:entry title="${%Docker Privileged Mode}" field="dockerPrivilegedMode" description="This will start the image using Docker's privileged mode.">
+                                        <f:checkbox field="dockerPrivilegedMode" value="${slaveInfo.containerInfo.dockerPrivilegedMode}" checked="false" />
```

Comment:
Also, does this setting maintain the checked value appropriately? I remember another PR doing this differently to maintain the checked value. You can test this by saving the checked state and then opening the config and seeing if the field is still checked. There was an issue with the volumes field for read only.